### DISCUSSION TITLE
Icon for dark and light

### DISCUSSION THREAD

**Discussion by gcailly** [jpanther/congo#321] [Q&A]
Hi !
I would like to display an "alert" box with custom icons.
So I put some black icons in `\assets\icons` and called `{{< alert "my_icon" >}}Text{{< /alert >}}`.
I works well.
But when I switch to dark theme, my icon seems out of place : it's black on dark. Il would like to use a lighter icon when I switch to dark theme.
Is there a method to do this easily ? I searched the documentation but didn't find it. I'm not very experienced with Hugo and HTML/CSS...

**Comment by jpanther** [comment]
Sorry for the delayed reply to this but have you included the `fill="currentColor"` in the `path` tag within the SVG asset? Double check against the icons that are included with the theme if you want an example. What this does is it tells the icon to follow the theme colours and will then appropriately switch between the light and dark appearances.
